test/spec/wdio/retries.js

```diff
@@ -0,0 +1,42 @@
+import { spawn } from 'child_process'
+import path from 'path'
+import Launcher from '../../../build/lib/launcher'
+
+const STANDALONE_FLAG = '--run-launcher-standalone'
+
+if (process.argv.includes(STANDALONE_FLAG)) {
```

Comment:
What is the standalone doing?

Reply:
I'll address your overall comment here as well. The temporary file approach you suggest was something I considered as opposed to spawning a separate process for the retry-tests and checking the output. The reason I went with the separate process is that it doesn't require modifying the filesystem, whereas the temporary file obviously does. It seemed that, since the tests already spawn child processes, the separate process approach would be guaranteed to always work, whereas there could be situations in which modifying the filesystem would not (readonly access, for whatever reason).

Here, a separate process is spawned that runs the retry-tests and checks the output of that process to determine whether or not this test should pass or fail.
Depending on whether or not the flag is set, this file either *spawns* the new process, or *is* the new process. This could just as easily have been done using two different files, but keeping everything together made sense.

If you prefer the temporary file approach, I'll change it, no problem.